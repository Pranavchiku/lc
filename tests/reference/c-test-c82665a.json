{
    "basename": "c-test-c82665a",
    "cmd": "lc --no-color --show-c {infile} -extra-arg=\"\"",
    "infile": "tests/test.cpp",
    "infile_hash": "9e5ef4c37b7e9e1f13f0082ce65459c1e596430887537eb8fd656b77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "c-test-c82665a.stdout",
    "stdout_hash": "f84227587720a88a15b04453a715208561f7a9d5f2bc0d217bc6185c",
    "stderr": "c-test-c82665a.stderr",
    "stderr_hash": "ddd9cdace524788d7316e5a86be7985b8064eee50598a8aa7bfd2eb5",
    "returncode": 0
}