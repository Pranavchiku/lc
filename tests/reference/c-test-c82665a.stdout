#include <inttypes.h>

#include <stdlib.h>
#include <stdbool.h>
#include <stdio.h>
#include <string.h>
#include <lfortran_intrinsics.h>

int32_t f(int32_t *x);



// Implementations
int32_t f(int32_t *x)
{
    int32_t __return_var;
    int32_t result;
    __return_var = result;
    return __return_var;
}

